org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+        /// <para>
+        /// Jobs require a public empty constructor so that the
+        /// scheduler can instantiate the class whenever it needs.
+        /// </para>
+        /// </summary>
+        public HubspotIntegrationPatching()
+        {
+        }
+
+        /// <summary>
+        /// Job that will run quick SQL queries on a schedule.
+        /// 
+        /// Called by the <see cref="IScheduler" /> when a
+        /// <see cref="ITrigger" /> fires that is associated with
+        /// the <see cref="IJob" />.
+        /// </summary>
+        public virtual void Execute(IJobExecutionContext context)
+        {
+            JobDataMap dataMap = context.JobDetail.JobDataMap;
+
+            // Get HS API Token
+            string attrKey = dataMap.GetString("AttributeKey");
+            string attrValue = GlobalAttributesCache.Get().GetValue(attrKey);
+            key = Encryption.DecryptString(attrValue);
+
+            var current_id = 0;
+
+            PersonService personService = new PersonService(new RockContext());
+
+            //Get Hubspot Properties in Rock Information Group
+            //This will allow us to add properties temporarily to the sync and then not continue to have them forever
+            var propClient = new RestClient("https://api.hubapi.com/crm/v3/properties/contacts?properties=name,label,createdUserId,groupName,options,fieldType");
+            propClient.Timeout = -1;
+            var propRequest = new RestRequest(Method.GET);
+            propRequest.AddHeader("Authorization", $"Bearer {key}");
+            IRestResponse propResponse = propClient.Execute(propRequest);
+            WriteToLog( "Is propResponse Null? : " + propResponse.IsNull().ToString() );
+            var props = new List<HubspotProperty>();
+            var tmbtProps = new List<HubspotProperty>();
+            var propsQry = JsonConvert.DeserializeObject<HSPropertyQueryResult>(propResponse.Content);
+            props = propsQry.results;
+            WriteToLog( "Is propsQry Null? : " + propsQry.IsNull().ToString() );
+            WriteToLog( "Is propsQry.results Null? : " + propsQry.results.IsNull().ToString() );
+            WriteToLog( "Is props Null? : " + props.IsNull().ToString() );
+            
+
+            //Filter to props in Rock Information Group (and Contact information group) 
+            props = props.Where(p => p.groupName == "rock_information" || p.groupName == "contactinformation").ToList();
+            //props = props.Where( p => p.groupName == "Contact information" ).ToList(); // - 6/26 sdlp - To update names, address, and phone
+            //tmbtProps = propsQry.results.Where(p => p.groupName == "rock_tmbt_information").ToList(); // ---------- Commented 6/25 sdlp
+            //Business Unit hs_all_assigned_business_unit_ids
+            //Save a list of the ones that are Rock attributes
+            //var attrs = props.Where(p => p.label.Contains("Rock Attribute ")).ToList(); // commented sdlp 7/15/24
```

Comment:
Please remove commented out code since it doesn't add to the readability of the code that is currently being used.